src/main/java/com/monumental/util/application/InitialDataLoadApplication.java

```diff
@@ -19,16 +27,23 @@
  * Then does validation on each CsvMonumentConverterResult's Monument before inserting it into the database
  * Finally, closes the CsvFileReader
  */
+@Configuration
+@SpringBootApplication
+@ComponentScan("com.monumental")
 public class InitialDataLoadApplication {
+
     public static void main(String[] args) {
-        String pathToDatasetCsv = "C:\\Users\\nickb\\Documents\\Initial Dataset.csv";
+        SpringApplication app = new SpringApplication(InitialDataLoadApplication.class);
+        // Don't start a web server - allows this application to run while the main Application is running
+        app.setWebApplicationType(WebApplicationType.NONE);
+        ConfigurableApplicationContext context = app.run(args);
+        String pathToDatasetCsv = "C:\\Users\\Ben\\Downloads\\Initial.csv";
         String csvRow;
         int rowCount = 0;
         ArrayList<CsvMonumentConverterResult> results = new ArrayList<>();
 
-        SessionFactoryService sessionFactoryService = new SessionFactoryService();
-        MonumentService monumentService = new MonumentService(sessionFactoryService);
-        TagService tagService = new TagService(sessionFactoryService);
+        MonumentService monumentService = context.getBean(MonumentService.class);
```

Comment:
Since we can't use autowire here, this is a way to get the services without using dependency injection

Reply:
I thought the point of making it a `@SpringBootApplication` meant we could Autowire?

Reply:
We can't use autowire because we're in a static context. You can only use autowire when your class itself is being autowired (`@Service`, `@Component`, etc.) and is in a non-static context (because that's how Spring does the dependency injection)